satellite/satellitedb/migrate.go

```diff
@@ -601,6 +601,41 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 					return nil
 				}),
 			},
+			{
+				Description: "Drops storagenode_storage_tally table, Renames accounting_raws to storagenode_storage_tally, and Drops data_type column",
+				Version:     17,
+				Action: migrate.SQL{
+					`DROP TABLE storagenode_storage_tallies CASCADE`,
+					`ALTER TABLE accounting_raws RENAME TO storagenode_storage_tallies`,
+					`ALTER TABLE storagenode_storage_tallies DROP COLUMN data_type`,
+				},
+			},
+			{
+				Description: "Retrofit storagenode_bandwidth_rollups into accounting_rollups since April 1, 2019",
+				Version:     18,
+				Action: migrate.SQL{
```

Comment:
can these migrations be one migration? and just add multiple actions statements to migration 17